frontend/src/Header.js

```diff
@@ -24,6 +26,8 @@ function Header() {
                     </Nav.Link>}
                     {checkPermission && user.role === "ROLE_SYSTEM_ADMIN" &&
                     <Nav.Link className={customersClass} href="/customers">Customers</Nav.Link>}
+                    {checkPermission && user.role === "ROLE_SYSTEM_ADMIN" &&
```

Comment:
checkPermission && user.role === "ROLE_SYSTEM_ADMIN" is in 2 places. Can be extracted to a single variable.